standalone-node/installation_scripts/standalone-vm-setup.sh

```diff
@@ -103,7 +103,8 @@ echo ""
 echo "Please see the installation status on VNC viewer.Enter $host_ip:1 on vnc viewer"
 sudo -E qemu-system-x86_64  \
   -m 4G   -enable-kvm  \
-  -cpu host \
+  -cpu host,+vmx \
```

Comment:
+vmx to -cpu host is appropriate for enabling hardware virtualization, but it may not be supported on all CPUs or configurations. May be add a comment at the start.